.travis.yml

```diff
@@ -32,15 +36,33 @@ matrix:
     env: TOXENV=py36
   - python: "3.7"
     env: TOXENV=py37
-  - python: "2.7"
-    env: TOXENV=py27-flake8
-  - python: "3.7"
-    env: TOXENV=py37-flake8
+  - os: osx
+    # No version of Python is available via virtualenv on OS X workers, see https://github.com/travis-ci/travis-ci/issues/2312
+    language: generic
+    env: TOXENV=py27
+  - os: osx
+    language: generic
+    env: TOXENV=py35
```

Comment:
Please note that I've skipped py34 since Python 3.4 is reaching End-of-Life on 2019-03-16: https://devguide.python.org/#status-of-python-branches